be/src/bench/orc_column_reader_bench.cpp

```diff
@@ -0,0 +1,357 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <benchmark/benchmark.h>
+#include <glog/logging.h>
+
+#include "formats/orc/column_reader.h"
+#include "formats/orc/memory_stream/MemoryInputStream.hh"
+#include "formats/orc/memory_stream/MemoryOutputStream.hh"
+#include "formats/orc/orc_chunk_reader.h"
+#include "formats/orc/orc_mapping.h"
+
+namespace starrocks {
+
+// Run command: ./be/build_Release/src/bench/output/orc_column_reader_bench 2>>/dev/null
+
+const static size_t benchmarkIterationTimes = 5;
```

Comment:
5 is relatively small.

Maybe we can use `MinTime()` to measure iterations of each setting.

>   // Set the minimum amount of time to use when running this benchmark. This
  // option overrides the `benchmark_min_time` flag.
  // REQUIRES: `t > 0` and `Iterations` has not been called on this benchmark.
  Benchmark* MinTime(double t);

Reply:
If we set larger iteration time, the later time-consuming will be shorter and shorter because of the cpu cache